cvat/apps/engine/migrations/0064_delete_wrong_labels.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from django.db import migrations
+from cvat.apps.engine.log import get_migration_logger
+
+def delete_wrong_labels(apps, schema_editor):
+    migration_name = os.path.splitext(os.path.basename(__file__))[0]
+    with get_migration_logger(migration_name) as log:
+        log.info('\nDeleting skeleton Labels without skeletons...')
+
+        Label = apps.get_model('engine', 'Label')
+        for label in Label.objects.all():
+            if label.type == "skeleton" and not hasattr(label, "skeleton"):
+                label.delete()
+
+        log.info('\nDeleting duplicate Labels...')
```

Comment:
Can such tasks be annotated already? If so, I suggest to rename the duplicate labels instead.

Reply:
The labels may have already been used, I added a rename for duplicates. I verified that it is not possible to add a skeletal annotation with repeated nested labels, so I suggest deleting such skeletons.

Reply:
Could you please also rename the file, the function, and update the log message?